docs/hive-integration/import-installed-cluster.md

```diff
@@ -0,0 +1,84 @@
+# Import Installed Cluster
+
+It is possible to import an existing installed OpenShift in order to be able to add more workers to it. (Day 2 operation)
+
+## CRDs
+
+Here are the needed CRDs to import an existing cluster:
+
+### InfraEnv
+
+* [InfraEnv](crds/infraEnv.yaml) or
+* [InfraEnv Late Binding](crds/infraEnvLateBinding.yaml)
+
+### NMState Config
+
+Optional network configuration.
+* [NMState Config](crds/nmstate.yaml)
+
+### Pull Secret
+
+* [PullSecret Secret](crds/pullsecret.yaml)
+
+### Cluster Image Set
+
+* [ClusterImageSet](crds/clusterImageSet.yaml)
+
+### AgentClusterInstall
+
+Configure the number of control planes in the `provisionRequirements` field to be greater than `1`.
```

Comment:
Why? Perhaps there is a room to elaborate here?

Is that because we do not allow adding day2 hosts to SNO clusters? If yes: maybe we can write it or link to another document that does?
Also: what would happen if the user set this to: `2`? will this pass validations?

Reply:
Yes, we do not allow adding day2 to SNO.
 if the user set this to 2, it will this pass validations since Day2 clusters is not looking at this parameter.

Reply:
Done